monero-outreach-docs/translations/pt-br/PSA/files/dontshare/dontshare_pt-br.md

```diff
@@ -0,0 +1,11 @@
+# Nunca compartilhe suas chaves privadas
+
+Seu Monero está seguro enquanto suas chaves privadas e semente mnemônica privada estiverem seguras. Suas chaves privadas e sua semente mnemônica privada são a única maneira pela qual você consegue acessar sua carteira e gastar/enviar seus XMR. Se você compartilhá-los, a pessoa com quem você os compartilhou (ou alguém que tenha a capacidade de roubá-los dessa pessoa) tem tanto acesso a sua carteira quanto você.
```

Comment:
Drop the repetition of "privada" here to make it more concise "Suas chaves privadas e sua semente mnemônica [...]"